awx/api/serializers.py

```diff
@@ -3640,7 +3682,11 @@ def _build_mock_obj(self, attrs):
                 setattr(mock_obj, field.name, getattr(self.instance, field.name))
         field_names = set(field.name for field in self.Meta.model._meta.fields)
         for field_name, value in list(attrs.items()):
-            setattr(mock_obj, field_name, value)
+            if field_name == 'execution_environment':
```

Comment:
I still can't figure out why the EE should have an exception case here.

Reply:
Removed.